app/api/v2/views/userview.py

```diff
@@ -29,6 +30,17 @@ def signup():
 
 
 class LoginUser():
+
+    @login.route('/login', methods=['POST'])
     def login():
         data = request.get_json()
-        
+        email = data['email']
+        password = data['password']
+
+        signeduser = UserModel()
+        signeduser.get_user_by_email(email)
+
+        return make_response(jsonify({
+            'status': 200,
+            'message': 'successfully logged in'
+        }), 200)
```

Comment:
Kindly handle the error that is returned here if a user tries to login without providing username or email or both.